test/Microsoft.Azure.Devices.Edge.Test/Module.cs

```diff
@@ -103,6 +103,11 @@ public async Task TempFilter()
         // Test Temperature Filter Function: https://docs.microsoft.com/en-us/azure/iot-edge/tutorial-deploy-function
         public async Task TempFilterFunc()
         {
+            if (OsPlatform.IsArm() && OsPlatform.Is64Bit())
+            {
+                Assert.Ignore("TempFilterFun is disabled for arm64 because azureiotedge-functions-filter does not exist for arm64");
```

Comment:
Missing the 'c' on the end of "TempFilterFunc"

Reply:
Fixed